src/providers/zig.rs

```diff
@@ -0,0 +1,80 @@
+use crate::nixpacks::{
+    nix::pkg::Pkg,
+    phase::{BuildPhase, InstallPhase, SetupPhase, StartPhase},
+};
+
+use super::Provider;
+
+pub struct ZigProvider;
+
+impl Provider for ZigProvider {
+    fn setup(
+        &self,
+        _app: &crate::nixpacks::app::App,
```

Comment:
nit: Can you please cleanup the imports and variable references so it isn't prefixed with `crate::nixpacks` everywhere. Like https://github.com/railwayapp/nixpacks/blob/main/src/providers/deno.rs#L20https://github.com/railwayapp/nixpacks/blob/main/src/providers/deno.rs#L20